src/console/plugin.ts

```diff
@@ -140,7 +140,7 @@ interface ICreateConsoleArgs extends JSONObject {
 /**
  * Activate the console extension.
  */
-function activateConsole(app: JupyterLab, services: IServiceManager, rendermime: IRenderMime, mainMenu: IMainMenu, inspector: IInspector, palette: ICommandPalette, pathTracker: IPathTracker, renderer: ConsolePanel.IRenderer, layout: ILayoutRestorer): IConsoleTracker {
+function activateConsole(app: JupyterLab, services: IServiceManager, rendermime: IRenderMime, mainMenu: IMainMenu, inspector: IInspector, palette: ICommandPalette, pathTracker: IPathTracker, renderer: ConsolePanel.IRenderer, restorer: IInstanceRestorer): IConsoleTracker {
```

Comment:
Because there are two plugins in this file, even though one of them uses an anonymous `activate` function, I still thought it might be better to leave the other one with a verbose name.